characterEntities/package-info.java

```diff
@@ -1,8 +0,0 @@
-/**
- * 
- */
-/**
- * @author Cathy
- *
- */
-package characterEntities;
```

Comment:
Not sure why package info was deleted, unless you deleted the entire characterEntity package. These are used for code documentation